packages/web3-eth-ens/src/abi/ens/ENSRegistry.ts

```diff
@@ -0,0 +1,395 @@
+/*
+This file is part of web3.js.
+
+web3.js is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+web3.js is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+export const ENSRegistryAbi = [
```

Comment:
We already have registry https://github.com/ChainSafe/web3.js/blob/4.x/packages/web3-eth-ens/src/abi/registry.ts , why do we need this new file?

Reply:
For running the tests, I needed the bytecode of the contracts so I compiled the ones mentioned in the headers. Initially, I wanted to be sure that I had the abi of the exactly same version.

However, when I tried to restore to the initial abis (the ones you mention) I was getting an error (TypeError: abis is not iterable). I'll keep investigating the error.